### PR TITLE
docs: add a pointer to community meetings in our docs.

### PR DESCRIPTION
**Pull request by klihub**
Add a section to the docs about community meetings and a pointer to the CNCF calendar with meeting details.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,12 +6,21 @@ the section below we ask that the following guidelines be considered when
...
 Slack channel should be used for discussion
 
+## Community Meetings
+
+We have regular community meetings in place to discuss CDI-related topics.
+The current cadence of the meetings is bi-weekly. You can find the meeting
+details in the [CNCF Community Meetings Calendar](https://www.cncf.io/calendar).
+The calendar entry also has a link to our community meeting agenda. The agenda
+has a summary of past meeting topics and discussions. It can also be freely
+edited by anyone to propose topics for upcoming meetings.
```
Comment (bart0sh) [inline CONTRIBUTING.md:22]: Should we also mention the name of the meeting as it shown at the link `[CNCF TAG Runtime Container Device Interface (COD) Working Group](https://tockify.com/cncf.public.events/detail/184/1719324000000?search=COD)`?

Comment (bart0sh) [reply CONTRIBUTING.md:22]: Pointing to the meeting notes https://docs.google.com/document/d/1gUgAMEThkRt4RJ7pA7ZbPPmIOX2Vb7fwH025MjfcTYU would also make sense, I think.

Comment (klihub) [reply CONTRIBUTING.md]: We discussed this and decided to deliberately not link directly to the meeting agenda, but instead mention that it is reachable via an up-to-date link through the meeting calendar entry. This way we don't need to worry about keeping the meeting agenda link up to date in the documentation if it ever changes.

And thanks for the tockify hint/link. I added now your suggested search link to our docs, so getting to the calendar should now be behind two clicks, the agenda behind 3 clicks, and no manual searching any more.

**Review by bart0sh — APPROVED** [review]
/lgtm

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -64,5 +76,5 @@ Then you just add a line to every git commit message:
 
 Use your real name (sorry, no pseudonyms or anonymous contributions.)
 
-If you set your `user.name` and `user.email` git configs, you can sign your
-commit automatically with `git commit -s`.
+If you set your `user.name` and `user.email` in your git config, you can
+sign your commit automatically with `git commit -s`.
```
Comment (elezar) [inline CONTRIBUTING.md:80]: Nit: I think it's `sign-off`. Sign implies the use of PGP keys for verified commits.

**Review by elezar — APPROVED** [review]
A minor nit, but it looks good!

**Pull request merged**